ResearchKit/Common/ORKQuestionStepViewController.m

```diff
@@ -756,8 +800,18 @@ - (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPa
 }
 
 - (CGFloat)heightForChoiceItemOptionAtIndex:(NSInteger)index {
-    ORKTextChoice *option = [(ORKTextChoiceAnswerFormat *)_answerFormat textChoices][index];
-    CGFloat height = [ORKChoiceViewCell suggestedCellHeightForShortText:option.text LongText:option.detailText inTableView:_tableView];
+    NSString *optionText = @"";
+    NSString *detailText = nil;
+    if ([_answerFormat isKindOfClass:[ORKMedicationAnswerFormat class]]) {
+        ORKMedicationCellText *medicationCellText = [self.medicationChoiceCellGroup medicationCellTextForRow:index];
```

Comment:
Could ORKMedicationCellText just be a ORKTextChoice? Are shortText and longText that much different from text and detailText?